### DISCUSSION TITLE
Divider vs StackDivider, why is one an HR?

### DISCUSSION THREAD

**Discussion by Bertg** [chakra-ui/chakra-ui#2537] [Q&A] [answered]
In our team we started building with Chakra some time ago, and we had a discussion about `<Divider />` and `<StackDivider />` when using in a `Stack`.

We noticed that the  `StackDivider` is a `div` while the `Divider` is an `hr`. 
Is there a reason for discrepancy, and how do we know which one we should be using?

Thanks!

**Comment by ljosberinn** [comment]
`Divider`is the regular `hr` element because thats most fitting for something dividing, that's the purpose of `hr`:
> The hr element represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section. (via [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr))

A `StackDivider` is used within a `Stack`, which most likely is just adding some spacing between elements, not signaling a `thematic break between paragraph-level elements`.

**Comment by Bertg** [comment, marked as answer]
@ljosberinn Would it be correct to rephrase it like this?

`StackDivider` is meant as a visual-only component. While `Divider` is both visual and semantic.
